recipes/pblaa/2.4.2/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pblaa" %}
+{% set version = "2.4.2" %}
+{% set laa_sha256 = "563a2f68c370f9e4f6eb3ac6656ee8a37be3abdae64124d7d077488dcb332172" %}
+{% set laagc_sha256 = "ec366dcbf1c6197e9baf4cb5147c8b41d5d76d860ed835ae6407c88d870205e0" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source: # must keep [linux] to prevent it being installed on mac os
```

Comment:
You can skip a build on OSX with: `skip: True  # [osx]` in the build/skip section.

Reply:
Thanks for the reminder. I have added `skip: True # [osx]` in the build/skip section.  I will remove this comment.